affine/__init__.py

```diff
@@ -236,14 +236,14 @@ def rotation(cls, angle, pivot=None):
         ca, sa = cos_sin_deg(angle)
         if pivot is None:
             return tuple.__new__(cls,
-                (ca, sa, 0.0,
-                -sa, ca, 0.0,
+                (ca, -sa, 0.0,
+                 sa, ca, 0.0,
                  0.0, 0.0, 1.0))
```

Comment:
It seems that this modification simply switches from clockwise convention to counter-clockwise. See https://en.wikipedia.org/wiki/Transformation_matrix#Rotation